mysql_database/TipTracker_Scripts/TipTracker_Shift_Data_Schema.sql

```diff
@@ -0,0 +1,73 @@
+USE sql5388964;
+
+-- The following command turns off the Safe Update checks so that your
+-- UPDATE and DELETE queries will work without restrictions.  You don't
+-- have to replicate this command anywhere else in your code.
+SET SQL_SAFE_UPDATES = 0;
+
+Drop table if exists `non_hourly_shift_details`;
+
+Drop table if exists `hourly_shift_details`;
+
+Drop table if exists `users`;
+
+Drop table if exists `employers`;
+ 
+CREATE TABLE `users` (
+  `user_id` int PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  `first_name` varchar(255) NOT NULL,
+  `last_name` varchar(255) NOT NULL,
+  `email` varchar(255) UNIQUE NOT NULL,
+  `password_hash` varchar(255) NOT NULL,
+  `timezone` varchar(255),
+  `created_at` timestamp
+);
+
+CREATE TABLE `employers` (
+  `employer_id` int AUTO_INCREMENT NOT NULL,
+  `employer_name` varchar(255) NOT NULL,
+  `industry` varchar(255) NOT NULL,
+  PRIMARY KEY (`employer_id`, `employer_name`)
+);
+
+CREATE TABLE `non_hourly_shift_details` (
+  `shift_id` int PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  `user_id` int NOT NULL,
+  `employer_id` int NOT NULL,
```

Comment:
More specifically, I thought we wanted to charge users a monthly subscription if they wanted to use multiple employers in the same platform.

Reply:
I think this is just a high-level version of the multiple employer functionality, so this will currently only link a single user to a single employer but later on, it will be able to link to multiple employers